shacl12-core/index.html

```diff
@@ -5691,6 +5692,194 @@ <h4>sh:property</h4>
 					</p>
 				</section>
 				
+				<section id="MemberShapeConstraintComponent">
+					<h4>sh:memberShape</h4>
+					<p>
+						<code>sh:memberShape</code> requires that each <a>value node</a> be a <a href="#syntax-rule-SHACL-list">SHACL list</a>,
+						and that each member of that list conforms to the given <a>node shape</a>.
+					</p>
+					<p>
+						<span class="component-class">Constraint Component IRI</span>: <code>sh:MemberShapeConstraintComponent</code>
+					</p>
+
+					<div class="parameters">Parameters:</div>
+					<table class="term-table">
+						<tr>
+							<th>Property</th>
+							<th>Summary and Syntax Rules</th>
+						</tr>
+						<tr>
+							<td><code>sh:memberShape</code></td>
+							<td>
+								The shape that all members of the <a href="#syntax-rule-SHACL-list">SHACL list</a> must conform to.
+								<span data-syntax-rule="memberShape-node">The value of <code>sh:memberShape</code> must be a <a>well-formed</a> <a>node shape</a>.</span>
+							</td>
+						</tr>
+					</table>
+					<div class="def def-text">
+						<div class="def-header">TEXTUAL DEFINITION</div>
+						<div class="def-text-body" data-validator="MemberShape">
+							Let <code>$memberShape</code> be a <a>value</a> of <code>sh:memberShape</code>.
+							Each <a>value node</a> <code>v</code> must be a <a href="#syntax-rule-SHACL-list">SHACL list</a> - a <a>failure</a> MUST be produced if <code>v</code> is not a SHACL list.
+							For each member <code>m</code> of the list <code>v</code>
+							a <a>failure</a> MUST be produced if the <a>conformance checking</a> of <code>m</code> against <code>$memberShape</code> produces a <a>failure</a>.
+							There must be at most one <a>validation result</a> for each list <code>v</code>.
+
+							The `sh:detail` property MUST be used to specify the validation results for each member <code>m</code> of the list that fails conformance checking.
+							The `sh:detail` property MUST also be used to specify when <code>v</code> is not a <a>SHACL list</a>.
```

Comment:
Nit: Shouldn't sh:detail be in an HTML code element, not a Markdown backtick?

Not-nit: Why the "MUST" for sh:detail "being used to specify the validation results..."?  I think I see that `sh:node` does not have guidance to adapt.  I agree this is at least SHOULD-strength, but what do others think about MUST-strength?

Also, the second line about not-a-SHACL-list feels like it needs illustration.  Can you in-page link to a demonstration?

Reply:
> Also, the second line about not-a-SHACL-list feels like it needs illustration. Can you in-page link to a demonstration?

Clarified offline that this was requesting a pointer to example of `sh:detail`. Added in https://github.com/w3c/data-shapes/pull/403/commits/7ff2a9a8c13811779f955788edcbe6811ac4a4a3